video-demo/imagination/scripts/capture_cli.py

```
@@ -0,0 +1,84 @@
+    def checkpoint(self,title,body,hint):
+        self.console.export_text(clear=True)
+        answer=next(self.answers)
+        with patch.object(self.console,'input',return_value=answer):
+            result=super().checkpoint(title,body,hint)
+        name={'The brief':'capture-checkpoint','The outline':'capture-outline','Chapter 1, read blind':'capture-chapter'}[title]
+        write_html(name,self.console)
+        self.captures.append({'title':title,'answer':answer,'capture':name})
+        return result
+
+spec=importlib.util.spec_from_file_location('demo_builder',SOURCE/'scripts/build_reader_demo.py')
+demo=importlib.util.module_from_spec(spec);spec.loader.exec_module(demo)
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)